cli/src/package.rs

```diff
@@ -32,6 +37,32 @@ pub enum Command {
         #[arg(long)]
         out_dir: PathBuf,
     },
+    /// Modify a local copy of the index, by adding a new version of a package.
+    ///
+    /// To actually publish to the global registry, you need to do a bunch more
+    /// steps. Eventually, we'll provide tooling to automate this.
+    ///
+    /// 1. Fork the nickel mine (github.com/nickel-lang/nickel-mine) on github.
+    /// 2. Clone your fork onto your local machine.
+    /// 3. The package you want to publish must be at the root of a git repository.
+    ///    `cd` into that repository (or supply `--manifest-path` in the next step).
+    /// 4. Run `nickel publish-to-local-index --index <directory-of-your-clone> --package-id github/you/your-package`
+    /// 5. You should see that your local machine's index was modified. Commit that modification.
+    /// 6. Push your package to the `you/your-package` repository on github. These
+    ///    names *must* match the package id in the index, and you must ensure that
+    ///    the version you push to github matches the SHA-1 hash in the index.
+    /// 7. Open a pull request to `github.com/nickel-lang/nickel-mind` to make your index
+    ///    modifications public.
+    PublishToLocalIndex {
```

Comment:
Nitpick: this sounds a bit long for a command name.

Reply:
Renamed to "publish".